### ISSUE TITLE
Rename entities?

### ISSUE THREAD

**Issue by svenove** [SAIC-iSmart-API/saic-home-assistant-addon#97]
The VIN is quite long and all entities get prefixed with the VIN. I can’t find a way to rename the entities to “mg5_xxxx” instead. Isn’t that possible?

**Comment by nanomad** [comment]
Not really, no. Some people have multiple MG cars so we need to support those scenarios.

You can safely rename the entity from HA UI though
